docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -16,7 +16,25 @@ Broadly speaking, there are three kinds of ScPCA data you might wish to work wit
     - These data are used for automated testing, but you can also use them while developing your module if smaller files helps make development more efficient.
     - You do not need an AWS account to download the test files, so you can use this data before you are granted full access to ScPCA data.
 
-## Accessing ScPCA Data
+
+!!! tip "Log into AWS CLI before running the download scripts"
```

Comment:
I think this tip is in the wrong place now. It should be under the S3 section, as you don't need these scripts for access through the Portal.